pkgs/by-name/li/libftdi1/package.nix

```diff
@@ -74,11 +74,14 @@ stdenv.mkDerivation rec {
   postPatch = ''
     substituteInPlace packages/99-libftdi.rules \
       --replace-fail 'GROUP="plugdev"' 'GROUP="ftdi"'
+
+    substituteInPlace packages/99-libftdi.rules \
+      --replace-fail 'GROUP="ftdi"' 'GROUP="ftdi", TAG+="uaccess"'
```

Comment:
If you have time/energy, maybe submit `TAG+="uaccess"` patch upstream? (And the required udev filename / order.)

Reply:
I've thought about it, but the libftdi project doesn't seem to be very active at the moment. Last merged commit is from end 2023 and some patches seem to be stalled. So I don't have lots of hopes that my change gets merged soon. I will keep it in mind though.